spec.html

```diff
@@ -40609,7 +40694,8 @@ <h1>Promise Resolve Functions</h1>
             1. Let _thenAction_ be _then_.[[Value]].
             1. If IsCallable(_thenAction_) is *false*, then
               1. Return FulfillPromise(_promise_, _resolution_).
-            1. Let _job_ be NewPromiseResolveThenableJob(_promise_, _resolution_, _thenAction_).
+            1. Let _thenJobCallback_ be HostMakeJobCallback(_thenAction_).
+            1. Let _job_ be NewPromiseResolveThenableJob(_promise_, _resolution_, _thenJobCallback_.[[Callback]]).
```

Comment:
This looks wrong? You surely need to preserve the actual `_thenJobCallback_`, not just extract the `[[Callback]]` component and discard that.

Reply:
Agh, was on autopilot, sorry.